grommet-leaflet-core/src/markers/Popup.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Box, Grommet } from 'grommet';
+import { hpe } from 'grommet-theme-hpe';
+import '../grommet-leaflet-reset.css';
```

Comment:
Assuming you weren't able to find a way not to need this?

Reply:
Correct, the `className` approach didn't work in the same way it did with clusters/single marker since there are a variety of elements that build up the popup.